weblate/addons/tasks.py

```diff
@@ -163,3 +163,25 @@ def setup_periodic_tasks(sender, **kwargs) -> None:
         cleanup_addon_activity_log.s(),
         name="cleanup-addon-activity-log",
     )
+
+
+@app.task(trail=True)
+def addon_change(sender, change, **kwargs) -> None:
```

Comment:
The task cannot receive `Change` object, as it's not something you can safely serialize. It needs to accept ID and fetch it from a database. Or actually a list of IDs (see previous comment) to fetch all changes from the database and then process them.